src/controllers/therapists.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, Response } from 'express';
+import getAllTherapist from '../services';
+
+const getAllTherapists = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { q, page } = req.query;
+    const pageNumber = parseInt(page as string, 2) || 1;
+    const limit = 8;
```

Comment:
let us declare this at separate file at config folder, and please UPPERCASE it as this is a constant ,
```js
const THERAPISTS_LIMIT = 8;
```